filesystem/.gitignore

```diff
@@ -1 +1,2 @@
 filesystem
+*.swp
```

Comment:
Put this in the top-level .gitignore instead of the one in this directory (or even better, put ignore entries related to your editor in your user-level ignore file so you don't have to put this in every repo you work with).